tests/tests_twobody/test_propagation.py

```diff
@@ -335,6 +335,12 @@ def test_propagate_to_date_has_proper_epoch():
     assert (ss1.epoch - final_epoch).sec == approx(0.0, abs=1e-6)
 
 
+@pytest.mark.filterwarnings(
```

Comment:
See my other comments

Reply:
@astrojuanlu again these ERFA warnings are generating upstream in astropy

Reply:
Yes, ERFA warnings are generated by Astropy, but it's probably our fault. See my comment above.